planning/mission_planner/README.md

```diff
@@ -25,11 +25,13 @@ In current Autoware.universe, only Lanelet2 map format is supported.
 
 ### Services
 
-| Name                                 | Type                                        | Description                       |
-| ------------------------------------ | ------------------------------------------- | --------------------------------- |
-| `/planning/routing/clear_route`      | autoware_adapi_v1_msgs::srv::ClearRoute     | route clear request               |
-| `/planning/routing/set_route_points` | autoware_adapi_v1_msgs::srv::SetRoutePoints | route request with pose waypoints |
-| `/planning/routing/set_route`        | autoware_planning_msgs::srv::SetRoute       | route request with HAD map format |
+| Name                                 | Type                                              | Description                       |
+| ------------------------------------ | ------------------------------------------------- | --------------------------------- |
+| `/planning/routing/clear_route`      | autoware_adapi_v1_msgs/srv/ClearRoute             | route clear request               |
+| `/planning/routing/set_route_points` | autoware_adapi_v1_msgs/srv/SetRoutePoints         | route request with pose waypoints |
+| `/planning/routing/set_route`        | autoware_planning_msgs/srv/SetRoute               | route request with HAD map format |
+| `~/srv/set_mrm_goal`                 | autoware_planning_msgs/srv/SetPoseWithUuidStamped | set emergency route               |
```

Comment:
`mrm_goal` is good for me since currently only one goal is sent for rerouting, but would you prefer `mrm_route`?

Reply:
@isamu-takagi how about this?:pray: I saw the comment ([tier4 internal link](https://star4.slack.com/archives/C4P0NSMB5/p1681120250209869?thread_ts=1678437689.314789&cid=C4P0NSMB5))

Reply:
ok, thanks
just a comment, that the name is `mrm_route` but the type is `PoseWithUuidStamped` is a little confusing, but it's ok for me.

Reply:
Renamed https://github.com/autowarefoundation/autoware_msgs/pull/59.

Reply:
Change service type https://github.com/autowarefoundation/autoware.universe/pull/3299/commits/aa320bc7f6b55f00e38f66ca973d25448f760a20.